src/arena.rs

```diff
@@ -261,6 +267,7 @@ impl<T> Arena<T> {
     }
 
     #[allow(unused_variables)]
+    #[deprecated]
```

Comment:
can we just remove these instead of marking deprecated?

Reply:
I did ;)